Formula/zlib.rb

```diff
@@ -38,17 +38,16 @@ def install
     system "./configure", "--prefix=#{prefix}"
     system "make", "install"
 
-    # Avoid rebuilds of dependants that hardcode this path.
+    # Avoid rebuilds of dependents that hardcode this path.
     inreplace lib/"pkgconfig/zlib.pc", prefix, opt_prefix
```

Comment:
> Also, fix a typo in a comment.

Isn't it an alternative spelling? Oh, but I'm the only one using this, okay.

```console
$ rg dependant 
Formula/libpng.rb
43:    # Avoid rebuilds of dependants that hardcode this path.

Formula/sqlite.rb
54:    # Avoid rebuilds of dependants that hardcode this path.

Formula/zlib.rb
41:    # Avoid rebuilds of dependants that hardcode this path.
```

Reply:
Ah, yes. British English.